api/__init__.py

```diff
@@ -0,0 +1,49 @@
+from typing import Dict, Iterator, Type, Tuple, List
+import pathlib
+import importlib
+import sys
+import logging
+
+import pydantic
+
+_logger = logging.getLogger(__name__)
+
+SCHEMAS_PATH = pathlib.Path(__file__).parent / "schemas"
+
+
+def load_public_schemas() -> List[Type[pydantic.BaseModel]]:
+    """Loads all pydantic Schemas for export.
+
+    If a class is prefixed with a '_' (i.e. _MySchema), it will not be exported
+    as a public schema.
+
+    Performs a bit of python magic to load all modules in the api/ folder and
+    find all subclasses of the base model.
```

Comment:
"the base model" => "pydantic.BaseModel" (always good to be as explicit as possible)